ROADMAP.md

```diff
@@ -29,44 +30,41 @@ Any dates listed below and the specific issues that will ship in a given milesto
-
-* Composition
-  * Bi-directional patching for status [#1639](https://github.com/crossplane/crossplane/issues/1639)
-  * Revision support for incremental upgrades [#1481](https://github.com/crossplane/crossplane/issues/1481)
-  * Support taking values from members to fill a connection secret [#1609](https://github.com/crossplane/crossplane/issues/1609)
+  * Pluggable webhooks support [#1152](https://github.com/crossplane/crossplane/issues/1152)
+  * Enhanced back-off across core controllers [#2157](https://github.com/crossplane/crossplane/issues/2157)
 
 * Providers
+  * AWS Bucket Late Init support [provider-aws#536](https://github.com/crossplane/provider-aws/pull/536)
   * Code Generation of Providers (towards 100% coverage)
     * AWS ACK Code Generation of the Crossplane provider-aws
-      * auto generate all available types that ACK supports from [aws-sdk-go/models/apis](https://github.com/aws/aws-sdk-go/blob/master/models/apis)
+      * support additional resources from [aws-sdk-go/models/apis](https://github.com/aws/aws-sdk-go/blob/master/models/apis)
+      * RDS DBCluster [provider-aws#546](https://github.com/crossplane/provider-aws/pull/546)
+      * SNS services [provider-aws#451](https://github.com/crossplane/provider-aws/issues/451)
+      * Generate reference and selector fields [provider-aws#484](https://github.com/crossplane/provider-aws/issues/484)
+      * Generate late-init function [provider-aws#491](https://github.com/crossplane/provider-aws/issues/491)
+      * Generate is-up-to-date function [provider-aws#490](https://github.com/crossplane/provider-aws/issues/490)
     * Azure Code Generation of the Crossplane provider-azure
       * auto generate available types from the Azure metadata
     * Clouds that don't have code gen pipelines
       * Wrap stateless Terraform providers [#262](https://github.com/crossplane/crossplane/issues/262)
 
-* Package Manager
-  * Conversion webhooks to support installing multiple API versions at the same time
-
 ### Under Consideration
 
 * General
-  * First-class multi-language support for defining `Compositions` and `Configuration` packages.
-    * https://github.com/crossplane-contrib/crossplane-cdk8s
-    * alpha release
-  * Per-namespace mapping of IRSA and workload identity for finer grained infra permissions in multi-tenant clusters
+  * Observe-only Crossplane resources (e.g. VPC, Subnet) for use in Compositions - [#1772](https://github.com/crossplane/crossplane/issues/1722)
+  * Iterate on and formalize the Crossplane Resource Model (XRM) [#2068](https://github.com/crossplane/crossplane/issues/2068)
+  * Managed resources can accept an array of resource references for cross-resource references (CRR)
+  * Per-namespace mapping of IRSA and workload identity for finer grained infra permissions in multi-tenant clusters [#2116](https://github.com/crossplane/crossplane/issues/2116)
   * Enhanced integration testing [#1033](https://github.com/crossplane/crossplane/issues/1033)
```

Comment:
we can probably move this to 1.2 since our LFX Mentorship program participant is starting work on it this week as per https://mentorship.lfx.linuxfoundation.org/project/2e2239cf-2964-434f-a5c9-676b857fc29a 💪

Reply:
awesome! 💪